src/SeleniumLibrary/keywords/formelement.py

```diff
@@ -231,16 +233,15 @@ def input_password(self, locator, password, clear=True):
 
         | Input Password | password_field | ${PASSWORD} |
 
-        Notice also that SeleniumLibrary logs all the communication with
-        browser drivers using the DEBUG level, and the actual password can
-        be seen there. Additionally, Robot Framework logs all arguments using
-        the TRACE level. Tests must thus not be executed using level below
-        INFO if the password should not be logged in any format.
+        Please notice that Robot Framework logs all arguments using
+        the TRACE level and tests must not be executed using level below
+        DEBUG if the password should not be logged in any format.
 
-        The `clear` argument is new in SeleniumLibrary 4.0
+        The `clear` argument is new in SeleniumLibrary 4.0. Hiding password
+        logging from Selenium logs is new in SeleniumLibrary 4.2.
         """
         self.info("Typing password into text field '%s'." % locator)
-        self._input_text_into_text_field(locator, password, clear)
+        self._input_text_into_text_field(locator, password, clear, True)
```

Comment:
Passing Boolean values (or None) as-is is not that good style. Better to name the argument like `disable_logging=True`.